core/src/main/scala/scala/pickling/Macros.scala

```diff
@@ -504,7 +499,7 @@ trait UnpicklerMacros extends Macro with UnpickleMacros with FastTypeTagMacros {
         if (sym.isModuleClass) {
           q"${sym.module}"
         } else if (cir.javaGetInstance) {
-          q"""java.lang.Class.forName(${tpe.toString}).getDeclaredMethod("getInstance").invoke(null)"""
+          q"""_root_.java.lang.Class.forName(${tpe.toString}).getDeclaredMethod("getInstance").invoke(null)"""
```

Comment:
I think it's just a matter of cowboy coding something fast.

FYI - I plan to heavily rework the macros in the coming months (assuming I have free time).